crates/bevy_gizmos/src/lib.rs

```diff
@@ -85,8 +89,25 @@ impl Plugin for GizmoPlugin {
             .add_plugins(RenderAssetPlugin::<LineGizmo>::default())
             .init_resource::<LineGizmoHandles>()
             .init_resource::<GizmoConfig>()
-            .init_resource::<GizmoStorage>()
-            .add_systems(Last, update_gizmo_meshes)
+            .init_resource::<GizmoStorage<Swap>>()
+            .init_resource::<GizmoStorage<Update>>()
+            .init_resource::<GizmoStorage<Fixed>>()
+            .add_systems(
+                Last,
+                (propagate_gizmos::<Fixed>, update_gizmo_meshes).chain(),
+            )
+            .add_systems(
+                RunFixedMainLoop,
+                (
+                    stash_default_gizmos.before(bevy_time::run_fixed_main_schedule),
+                    pop_default_gizmos.after(bevy_time::run_fixed_main_schedule),
+                ),
+            )
+            .add_systems(
+                FixedFirst,
+                (clear_gizmo_context::<Fixed>, clear_gizmo_context::<Update>).chain(),
+            )
+            .add_systems(FixedLast, collect_default_gizmos::<Fixed>)
```

Comment:
This is so that anyone directly using `Gizmos<Fixed>` won't be surprised when it just doesn't appear. Otherwise anyone directly calling it unless FixedMain doesn't run or they run the system after FixedMain won't see anything.